Cesium3DTiles/include/Cesium3DTiles/Tileset.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <Cesium3DTiles/Library.h>
+#include <Cesium3DTiles/TilesetSpec.h>
+
+#include <glm/mat4x4.hpp>
+
+#include <functional>
+
+namespace Cesium3DTiles {
+
+/** @copydoc TilesetSpec */
+struct CESIUM3DTILES_API Tileset : public TilesetSpec {
+  Tileset() = default;
+
+  /**
+   * @brief A callback function for {@link forEachTile}.
+   */
+  typedef void ForEachTileCallback(
+      Tileset& tileset,
+      Tile& tile,
+      const glm::dmat4& transform);
+
+  /**
+   * @brief Apply the given callback to all tiles.
```

Comment:
I think it's worth mentioning in the doc that this will only invoke the callback for explicit tiles. It won't follow external tilesets, and it won't reify implicit tiles.

Reply:
Thanks for the suggestion, added a note.